pydy/codegen/tests/test_c_code.py

```diff
@@ -105,7 +105,17 @@ def test_generate_comma_lists(self):
                     'v0(t), v1(t), v2(t), v3(t), v4(t), v5(t)',
                     'f5(t), f2(t), f3(t), f0(t), f4(t), f1(t)')
 
-        assert self.generator.comma_lists() == expected
+        actual = self.generator.comma_lists()
+        assert type(actual) == type(expected)
+        assert len(actual) == len(expected)
+        for arg, expected_arg in zip(actual, expected):
+            if expected_arg == expected[-1]:
+                arg_vals = set(x.strip() for x in arg.split(','))
```

Comment:
The order of the strings is important here, so I'm not sure this does the right thing anymore.